From 2a888ba4043b536a30c3774f22c77308559916f0 Mon Sep 17 00:00:00 2001
From: Joe Zhang <joe.zhang@technexion.com>
Date: Thu, 9 Jan 2025 15:14:01 +0800
Subject: [PATCH 1/2] codec: wm8960: reduce annoying log in debian

For each user login, this line will report for 12 times.
---
 sound/soc/codecs/wm8960.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/wm8960.c b/sound/soc/codecs/wm8960.c
index 0c942089e2f9..368e8d3eaa49 100644
--- a/sound/soc/codecs/wm8960.c
+++ b/sound/soc/codecs/wm8960.c
@@ -1370,7 +1370,7 @@ static int wm8960_set_dai_sysclk(struct snd_soc_dai *dai, int clk_id,
 		}
 
 		wm8960->freq_in = freq_in;
-		dev_info(component->dev, "Using codec PLL. MCLK rate %d Hz\n", wm8960->freq_in);
+		dev_dbg(component->dev, "Using codec PLL. MCLK rate %d Hz\n", wm8960->freq_in);
 	}
 
 	switch (clk_id) {
-- 
2.34.1

